scripts/pylint_extensions.py

```diff
@@ -1252,7 +1252,7 @@ def process_module(self, node):
                     'no-space-at-beginning', line=line_num + 1)
 
             # Check if comment contains version info or data type.
-            if not previous_line.startswith(b'#'):
+            if not previous_line.startswith(b'#') and line.startswith(b'#'):
```

Comment:
Hm. Could you please explain the rationale/analysis behind this change, in detail? It is currently unclear to me how this ties into the lint failures.

Thanks!

Reply:
@seanlip Previously it was checking all the lines in a file but now it will only check comments for data structures(e.g. str, int, etc). And will avoid non-ASCII characters because there are no non-ASCII characters present in the comments.

Reply:
Right -- but why does that cause the lint checks to come to a complete stop? That seems like strange behaviour and we should understand that instead.

Also, the lint checks are still failing FYI.

Reply:
Maybe because we are using pylint-1.9.4 which runs on python2 and try to run it on python3 files which might be the cause of these errors as explained [here](https://github.com/AtomLinter/linter-pylint/issues/233#issuecomment-356304713)